services/tables/src/main/java/com/linkedin/openhouse/tables/api/validator/impl/OpenHouseTablesApiValidator.java

```diff
@@ -124,15 +124,19 @@ private List<String> validateUUIDForReplicaTable(
   }
 
   private void validatePolicies(CreateUpdateTableRequestBody createUpdateTableRequestBody) {
-    if (!policiesSpecValidator.validate(
-        createUpdateTableRequestBody.getPolicies(),
-        createUpdateTableRequestBody.getTimePartitioning(),
+    TableUri tableUri =
         TableUri.builder()
             .tableId(createUpdateTableRequestBody.getTableId())
             .clusterId(createUpdateTableRequestBody.getClusterId())
             .databaseId(createUpdateTableRequestBody.getDatabaseId())
-            .build(),
-        createUpdateTableRequestBody.getSchema())) {
+            .build();
+    if (!policiesSpecValidator.validate(
+            createUpdateTableRequestBody.getPolicies(),
+            createUpdateTableRequestBody.getTimePartitioning(),
+            tableUri,
+            createUpdateTableRequestBody.getSchema())
+        || !policiesSpecValidator.validateReplication(
```

Comment:
Replication being part of policies can be validated within a global polices spec validation process. i.e `policiesSpecValidator.validateReplication` call can move inside `policiesSpecValidator.validate` to this part cleaner.